backend/loi-service/pom.xml

```diff
@@ -93,8 +93,6 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <scope>runtime</scope>
-            <version>${postgresql.version}</version>

```

Comment:
Why did you remove the version? Keep it.

Reply:
The version we removed for the fix of additional details was not coming in proper format.